### PR TITLE
Add translation support

### PR DESCRIPTION
**Pull request by Pross**
Translation fixes for #50

### CODE REVIEW

**Comment by Pross** [comment]
Testing with pig-latin
screen shot 2016-07-20 at 22 55 49

**Comment by Pross** [comment]
Just working on qunit test...

**Comment by octalmage** [comment]
This is awesome! Thanks a ton.

File: src/js/run.js [hunk 1]
```diff
@@ -230,11 +230,11 @@ function displayReport( response ) {
 
 	// Display global compatibility status.
 	if ( compatible ) {
-		$( '#standardMode' ).prepend( '<h3>Your WordPress install is PHP ' + test_version + ' compatible.</h3>' );
+		$( '#standardMode' ).prepend( '<h3>Your WordPress install is PHP ' + test_version + ' ' + window.wpephpcompat.compatible + '.</h3>' );
```
Comment (octalmage) [inline src/js/run.js]: The first part of this currently isn't translatable.

File: wpengine-phpcompat.php [hunk 2]
```diff
@@ -286,12 +302,12 @@ function settings_page() {
...
 					{{update}}<br>
-					<textarea style="display: none; white-space: pre;">{{logs}}</textarea><a class="view-details">view details</a>
+					<textarea style="display: none; white-space: pre;">{{logs}}</textarea><a class="view-details"><?php esc_attr_e( 'view details', 'php-compatibility-checker' ); ?></a>
 				</div>
 				<?php $update_url = site_url( 'wp-admin/update-core.php' , 'admin' ); ?>
-				<div style="float:right;">{{#if updateAvailable}}<div class="badge wpe-update"><a href="<?php echo esc_url( $update_url ); ?>">Update Available</a></div>{{/if}}{{#if warnings}}<div class="badge warnings">{{warnings}} Warnings</div>{{/if}}{{#if errors}}<div class="badge errors">{{errors}} Errors</div>{{/if}}</div>
+				<div style="float:right;">{{#if updateAvailable}}<div class="badge wpe-update"><a href="<?php echo esc_url( $update_url ); ?>"><?php esc_attr_e( 'Update Available', 'php-compatibility-checker' ); ?></a></div>{{/if}}{{#if warnings}}<div class="badge warnings">{{warnings}} Warnings</div>{{/if}}{{#if errors}}<div class="badge errors">{{errors}} <?php esc_attr_e( 'Errors', 'php-compatibility-checker' ); ?></div>{{/if}}</div>
```
Comment (octalmage) [inline wpengine-phpcompat.php]: I think "Warnings" needs to be updated also!

File: src/js/run.js (hunk 1 above)
Comment (stevenkword) [inline src/js/run.js]: +1

File: src/wpephpcompat.php [hunk 3]
```diff
@@ -190,14 +190,14 @@ public function start_test() {
 			$this->debug_log( 'Processing: ' . $directory->post_title );
 
 			// Add the plugin/theme name to the results.
-			$scan_results .= 'Name: ' . $directory->post_title . "\n\n";
+			$scan_results .= __( 'Name: ', 'php-compatibility-checker' ) . $directory->post_title . "\n\n";
```
Comment (octalmage) [inline src/wpephpcompat.php]: Here we're translating `Name: `, but in the JavaScript we're just doing `Name`. Could these be the same translation?

Comment (Pross) [inline src/wpephpcompat.php]: Indeed, i'll change that today

**Comment by octalmage** [comment]
Sorry for the delay! This works great for me.

**Pull request merged**